{"url":"https://api.github.com/repos/owncloud/core/issues/2116","labels_url":"https://api.github.com/repos/owncloud/core/issues/2116/labels{/name}","comments_url":"https://api.github.com/repos/owncloud/core/issues/2116/comments","events_url":"https://api.github.com/repos/owncloud/core/issues/2116/events","html_url":"https://github.com/owncloud/core/issues/2116","id":11683376,"number":2116,"title":"cron service fails","user":{"login":"internalkernel","id":1035876,"avatar_url":"https://gravatar.com/avatar/5d5430fbb38d543b1c46d785c83ef92d?d=https%3a%2f%2fidenticons.github.com%2f949cde3167cea5c5598fca7071d347a8.png&r=x","gravatar_id":"5d5430fbb38d543b1c46d785c83ef92d","url":"https://api.github.com/users/internalkernel","html_url":"https://github.com/internalkernel","followers_url":"https://api.github.com/users/internalkernel/followers","following_url":"https://api.github.com/users/internalkernel/following{/other_user}","gists_url":"https://api.github.com/users/internalkernel/gists{/gist_id}","starred_url":"https://api.github.com/users/internalkernel/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/internalkernel/subscriptions","organizations_url":"https://api.github.com/users/internalkernel/orgs","repos_url":"https://api.github.com/users/internalkernel/repos","events_url":"https://api.github.com/users/internalkernel/events{/privacy}","received_events_url":"https://api.github.com/users/internalkernel/received_events","type":"user","site_admin":false},"labels":[{"url":"https://api.github.com/repos/owncloud/core/labels/bug","name":"bug","color":"fc2929"}],"state":"closed","assignee":null,"milestone":null,"comments":21,"created_at":"2013-03-05t19:57:10z","updated_at":"2013-09-24t07:41:20z","closed_at":"2013-04-16t11:01:48z","pull_request":{"html_url":null,"diff_url":null,"patch_url":null},"body":"### expected behaviour\r\ncron service works\r\n\r\n### actual behaviour\r\ncron service fails with error\r\n\r\n### steps to reproduce\r\n1. set cron job in admin control panel\r\n2. add entry to crontab\r\n3. watch the magic\r\n\r\n### server configuration\r\noperating system: ubuntu 12.04\r\n\r\nweb server: apache 2.22\r\n\r\ndatabase: mysql 5.5.29\r\n\r\nphp version: 5.3.10\r\n\r\nowncloud version: 5.0 rc2\r\n\r\n### logs\r\n#### web server error log\r\n```\r\nphp fatal error: call to a member function getroot() on a non-object in /var/www/owncloud/lib/files/filesystem.php on line 313\r\nunexpected error!\r\n```\r\n### crontab entry\r\n```\r\n5 * * * *    /usrbin/php    /var/www/owncloud/cron.php\r\n```\r\n\r\n#### other attempts\r\ni've also tried as a web service with the following crontab command: \r\n```\r\nwget -o - -v --no-check-certificate https://domain.com/cron.php\r\n```\r\nwhich produces the following in the owncloud error logs: \r\n```\r\n{\"app\":\"php\",\"message\":\"call to a member function getroot() on a non-object at \\/var\\/www\\/owncloud\\/lib\\/files\\/filesystem.php#313\",\"level\":4,\"time\":1362496082}\r\n```","closed_by":{"login":"kabum","id":245432,"avatar_url":"https://gravatar.com/avatar/a1dea19f88f38ea0b5b5b4d19d5b3107?d=https%3a%2f%2fidenticons.github.com%2f53d8cbb3dbc087a02951dec47438ccdf.png&r=x","gravatar_id":"a1dea19f88f38ea0b5b5b4d19d5b3107","url":"https://api.github.com/users/kabum","html_url":"https://github.com/kabum","followers_url":"https://api.github.com/users/kabum/followers","following_url":"https://api.github.com/users/kabum/following{/other_user}","gists_url":"https://api.github.com/users/kabum/gists{/gist_id}","starred_url":"https://api.github.com/users/kabum/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/kabum/subscriptions","organizations_url":"https://api.github.com/users/kabum/orgs","repos_url":"https://api.github.com/users/kabum/repos","events_url":"https://api.github.com/users/kabum/events{/privacy}","received_events_url":"https://api.github.com/users/kabum/received_events","type":"user","site_admin":false}}